.github/workflows/docker-helm.yml

```diff
@@ -11,7 +11,7 @@ env:
   DOCKERHUB_LUNETTES_REPO: lunettes/lunettes
   DOCKERHUB_GRAFANA_REPO: lunettes/grafana
   # Plugins to be installed.
-  GRAFANA_PLUGINS: yesoreyeram-infinity-datasource marcusolsson-json-datasource volkovlabs-form-panel marcusolsson-dynamictext-panel
+  GRAFANA_PLUGINS: yesoreyeram-infinity-datasource marcusolsson-json-datasource marcusolsson-dynamictext-panel
```

Comment:
It's a bug that doesn't take into account the version of the plugin. Maybe you could replace the shell separator with a comma, i.e. IFS=',', so that the variable that the for loop iterates over can use pluginname + space + version.

```yaml
GRAFANA_PLUGINS: yesoreyeram-infinity-datasource,marcusolsson-json-datasource,marcusolsson-dynamictext-panel,volkovlabs-form-panel 3.1.0
```

Reply:
+1, for future management of plugins, can you please add versions to volkovlabs-form-panel